core/jvm/src/test/scala/scalaz/zio/QueueSpec.scala

```diff
@@ -173,6 +174,9 @@ class QueueSpec(implicit ee: ExecutionEnv) extends AbstractRTSSpec with AroundTi
       e65
     )}
     """
+  }
+
+  import scalaz.zio.duration._
```

Comment:
Let's move this import to the top of the file.

Reply:
Then we'll have the ambiguous implicit conversion problem with `scala.concurrent.duration`.